qt/scientific_interfaces/Indirect/IndirectFittingModel.cpp

```diff
@@ -514,7 +516,8 @@ void IndirectFittingModel::addNewWorkspace(MatrixWorkspace_sptr workspace,
 }
 
 void IndirectFittingModel::removeWorkspace(std::size_t index) {
-  removeFittingData(index);
+  if (m_fittingData.size() > index)
```

Comment:
`>=`?

Reply:
@mantid-roman I think it should remain as `>` because the `index` starts at 0 and the largest possible index will be `size - 1`?

Reply:
That's right. I was just confused by the order of the operands.